qa-automation/readme.md

```diff
@@ -19,29 +19,37 @@ Your task is to write some end-to-end tests with [Cypress](https://www.cypress.i
 ## Acceptance Criteria
 
 1. Write a first end-to-end test that
-   1. Starts on the [landing page](https://www.binance.com/en)
-   1. Click the link [view more market](https://www.binance.com/en/markets)
-   1. Clicks the ETH/BTC button leading to [Pair trading view](https://www.binance.com/en/trade/ETH_BTC)
-   1. Verifies that all important information is loaded, e.g. trading chart, limit, market, stop limit input boxes, etc. (leaving this purposefully open to see what you think makes sense to test)
-   
-1. Write a second end-to-end test that
-   1. Starts on the [Pair trading view](https://www.binance.com/en/trade/ETH_BTC)
-   1. Enters some valid values into the Buy ETH form
-   1. Make sure the Total field has the correct value (Total = price \* Amount)
-   1. Verifies that you can't submit the form since you are not logged in
-   
-1. Write a data last end-to-end (WebSockets) test that
-   1. Starts on the [Pair trading view](https://www.binance.com/en/trade/ETH_BTC)
-   1. Under the hood, the UI uses WebSockets to fetch tickers
-   1. Verifies that the data is loaded in the tickers,
-   1. Verifies there is a consistent stream of data.
-   1. Verifies the time it takes the socket connection to connect is less than 1 second
+   1. Starts on the [Tradeling home page](https://tradeling.com/)
```

Comment:
make them use the dev link or any other public website, they can't use our products to write automation test.

Reply:
sure thanks for the feedback @devypt

Reply:
Binance website link included